### PR TITLE
Corrects white stripe on bottom of the page

### PR DESCRIPTION
**Pull request by joaojusto**
[Fix this bug](https://trello.com/c/QaGvjM7k/82-white-border-on-bottom-of-the-pages)

### CODE REVIEW

File: app/assets/stylesheets/components/_page.css.sass [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .Page
-  display: inline-block // needed to clearfix content
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_page.css.sass]: Why are you removing this?

Comment (joaojusto) [inline app/assets/stylesheets/components/_page.css.sass]: Because this is what was causing the issue

Comment (gabrielpoca) [inline app/assets/stylesheets/components/_page.css.sass]: There was a purpose to this, I hope it doesn't break.

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**